lib/MusicBrainz/Server/Translation.pm

```diff
@@ -137,11 +138,12 @@ sub set_language
     my $set_lang_munge = $set_lang =~ s/_([A-Z]{2})/-\L$1/r;
     my $set_lang_nocountry = $set_lang =~ s/_[A-Z]{2}//r;
     # Change en_AQ back to en-aq to compare with MB_LANGUAGES
-    if (any { $set_lang eq $_ || $set_lang_munge eq $_ } DBDefs->MB_LANGUAGES) {
+    if (contains_string([ DBDefs->MB_LANGUAGES ], $set_lang) ||
+        contains_string([ DBDefs->MB_LANGUAGES ], $set_lang_munge)) {
```

Comment:
if we have to check multiple strings I'd just keep the `any`, since the new code isn't much clearer, just slower